src/app/Scenes/BottomTabs/BottomTabsButton.tsx

```diff
@@ -84,21 +86,28 @@ export const BottomTabsButton: React.FC<BottomTabsButtonProps> = ({
     >
       <View style={{ flex: 1 }}>
         <ProgressiveOnboardingFindSavedArtwork tab={tab}>
-          <IconWrapper>
-            <BottomTabsIcon tab={tab} state="inactive" />
-          </IconWrapper>
-          <IconWrapper>
-            <Animated.View
-              style={{
-                opacity: activeProgress,
-                backgroundColor: "white",
-                width: ICON_WIDTH,
-                height: ICON_HEIGHT,
-              }}
-            >
-              <BottomTabsIcon tab={tab} state="active" />
-            </Animated.View>
-          </IconWrapper>
+          <Flex flex={1} alignItems="center">
+            <Flex flex={1}>
+              <IconWrapper>
+                <BottomTabsIcon tab={tab} state="inactive" />
+              </IconWrapper>
+              <IconWrapper>
+                <Animated.View
+                  style={{
+                    opacity: activeProgress,
+                    backgroundColor: "white",
+                    width: ICON_WIDTH,
+                    height: ICON_HEIGHT,
+                  }}
+                >
+                  <BottomTabsIcon tab={tab} state="active" />
+                </Animated.View>
+              </IconWrapper>
+            </Flex>
+            <Flex height={BOTTOM_TABS_TEXT_HEIGHT}>
```

Comment:
Just to make the code easier and more pleasant to read :)

```suggestion
            </Flex>

            <Flex height={BOTTOM_TABS_TEXT_HEIGHT}>
```